### PR TITLE
Add support to fulfillment events

### PR DESCRIPTION
**Pull request by volmer**
This PR adds support to fulfillment events. Fulfillment events are simple resources that can be CRUDed under `/admin/orders/:order_id/fulfillments/:fulfillment_id/events`.

Please review @christianblais @csaunders

### CODE REVIEW

File: test/fixtures/fulfillment_event.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "fulfillment_event": {
+    "created_at": "2013-11-01T16:06:08-04:00",
+    "updated_at": "2013-11-01T16:06:08-04:00",
+    "id": 334455,
+    "fulfillment_id": 255858046,
+    "order_id": 450789469,
+    "status": "in_transit"
```
Comment (christianblais) [inline test/fixtures/fulfillment_event.json]: Shouldn't you have a `happened_at` and maybe a `estimated_delivery_at` here?

**Comment by christianblais** [comment]
Works with me. Let's not merge before we get Tobi's approval though.

File: test/fixtures/fulfillment_event.json (hunk 1 above)
Comment (volmer) [inline test/fixtures/fulfillment_event.json]: Done.

**Comment by csaunders** [comment]
Do we have an API team anymore? Are we planning on making this public?

The code seems fine, though I'm mostly concerned about whether this is sufficient enough for what our users would need.

@jamesmacaulay might be interested in this as well.

**Comment by volmer** [comment]
> Do we have an API team anymore? Are we planning on making this public?

Yes, the docs will be released as soon as we merge this.

> The code seems fine, though I'm mostly concerned about whether this is sufficient enough for what our users would need.

For now it is, as far as I can see. Would you have any scenarios in mind not covered by these changes?

**Comment by christhomson** [comment]
:heart: this!

**Pull request merged**